notebooks/src/covid-19.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>Now that this is just for cases, the name and docstring of the function no longer make sense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>